src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -169,7 +169,7 @@
                   <div class="content-title">
                     <p>
                       {{ 'lbl.PreviousVersions'|trans|ucfirst }}
-                      <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpRevisions'|trans|ucfirst }}"></abbr>
+                      <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpRevisions'|trans|ucfirst }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320